zproject/prod_settings_template.py

```diff
@@ -49,12 +49,12 @@
 # accessing your server by other hostnames, list them here.
 #
 # Note that these should just be hostnames, without port numbers.
-#ALLOWED_HOSTS = ['zulip-alias.example.com', '192.0.2.1']
+# ALLOWED_HOSTS = ['zulip-alias.example.com', '192.0.2.1']
```

Comment:
We need to exclude this file, since we're using comments without a space to distinguish variables from text.

(It'd be fine to swap the strings, though)

Reply:
We could use `#` vs. `##` for that distinction (Black preserves `##`)?